index.html

```diff
@@ -216,9 +223,9 @@ <h2>Terminology</h2>
 [[HR-TIME-2]]. For example, the <a data-cite=
 "NAVIGATION-TIMING-2#dom-PerformanceNavigationTiming-startTime">start
 of navigation of the document</a> occurs at time 0.</p>
-<p>The term <dfn>current time</dfn> refers to the number of
-milliseconds since the start of navigation of the document until
-the current moment in time.</p>
+<p>The term <dfn>current time</dfn> refers to the result of running 
```

Comment:
I think we'd be better off eliminating this dfn, and call the algorithm directly with the current global object from the few places in the processing model that are calling it. 

The reason for that is that we need to call it with a specific global object, and this dfn doesn't give us that. Does that make sense?

Reply:
Sounds good and also seems more robust since I have observed that the term `current time` is mostly used in algorithmic steps in this spec.

So would replacing occurences like:
`record the <a>current time</a> in <a>startTime</a>`
with
`record the result of <a data-cite="HR-TIME-ED#dfn-current-high-resolution-time">current high 
resolution time</a> where |current global| is {{Window}} in <a>startTime</a>`
fit the case ?

Reply:
`{{Window}}` is a reference to the Window object, but not to a specific window.
You probably want to reference the [current global object](https://html.spec.whatwg.org/multipage/webappapis.html#current-global-object)